app/views/layouts/login.html.erb

```diff
@@ -1,8 +1,6 @@
 <%= content_for(:content) do %>
   <%= notifications %>
-  <div class="login-page" >
```

Comment:
This class is also used in the external logout scss

Reply:
good catch ! updated that file too.